### ISSUE TITLE
Uncaught Error: Cannot find module 'uuid'

### ISSUE THREAD

**Issue by eitanakash** [atom-ide-community/atom-script#1614]

**Comment by aceking007** [comment]
1. Install script plugin in Atom
2. Error pops up after installation

**Atom**: 1.34.0 x64
**Electron**: 2.0.16
**OS**: Microsoft Windows 10 Pro
**Thrown From**: [script](https://github.com/rgbkrk/atom-script) package 3.18.1


### Stack Trace

Uncaught Error: Cannot find module 'uuid'

```
At module.js:545

Error: Cannot find module 'uuid'
    at Module._resolveFilename (module.js:543:15)
    at Module._resolveFilename (~/AppData/Local/atom/app-1.34.0/resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.get_Module._resolveFilename (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:11:154667)
    at Module.require (/app.asar/static/index.js:40:43)
    at require (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:11:145974)
    at /packages/script/lib/grammar-utils.js:6:24)
    at /packages/script/lib/grammar-utils.js:109:2)
    at Module.get_Module._compile (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:11:146684)
    at Object.value [as .js] (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:11:150231)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:504:12)
    at Function.Module._load (module.js:496:3)
    at Module.require (/app.asar/static/index.js:47:45)
    at require (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:11:145974)
    at /packages/script/lib/grammars/index.coffee:5:45)
    at /packages/script/lib/grammars/index.coffee:4:1)
    at /packages/script/lib/grammars/index.coffee:4:1)
    at Module.get_Module._compile (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:11:146684)
    at Object.value [as .coffee] (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:11:150231)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:504:12)
    at Function.Module._load (module.js:496:3)
    at Module.require (/app.asar/static/index.js:47:45)
    at require (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:11:145974)
    at /packages/script/lib/grammars.js:9:28)
    at /packages/script/lib/grammars.js:1:13)
    at Module.get_Module._compile (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:11:146684)
    at Object.value [as .js] (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:11:150231)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:504:12)
    at Function.Module._load (module.js:496:3)
    at Module.require (/app.asar/static/index.js:47:45)
    at require (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:11:145974)
    at /packages/script/lib/code-context-builder.js:5:41)
    at /packages/script/lib/code-context-builder.js:8:40)
    at Module.get_Module._compile (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:11:146684)
    at Object.value [as .js] (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:11:150231)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:504:12)
    at Function.Module._load (module.js:496:3)
    at Module.require (/app.asar/static/index.js:47:45)
    at require (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:11:145974)
    at /packages/script/lib/script.js:3:43)
    at /packages/script/lib/script.js:217:2)
    at Module.get_Module._compile (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:11:146684)
    at Object.value [as .js] (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:11:150231)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:504:12)
    at Function.Module._load (module.js:496:3)
    at Module.require (/app.asar/static/index.js:47:45)
    at require (internal/module.js:11:18)
    at customRequire (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:1:641362)
    at Package.requireMainModule (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:11:3029314)
    at Package.activateConfig (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:11:3018673)
    at PackageManager.t.exports.PackageManager.packageHasSettings (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:11:892672)
    at PackageCard.hasSettings (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:14:2932592)
    at PackageCard.updateSettingsState (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:14:2922872)
    at PackageCard.updateInterfaceState (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:14:2922695)
    at ~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:14:2930825
    at ~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:14:2932843
    at Function.simpleDispatch (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:11:1172318)
    at Emitter.emit (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:11:1173759)
    at PackageManager.t.exports.PackageManager.emitPackageEvent (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:11:901852)
    at ~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:11:899261
    at r (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:11:893791)
    at o (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:14:1032547)
    at ChildProcess.n.process.on.e (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:14:1032753)
    at emitTwo (events.js:126:13)
    at ChildProcess.emit (events.js:214:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
```

### Commands

```
     -5:55.2.0 core:backspace (input.hidden-input)
```

### Non-Core Packages

```
script 3.18.1 
```

**Comment by aminya** [comment]
Fixed in #2339
